app/src/main/java/org/oppia/app/player/exploration/ExplorationActivityPresenter.kt

```diff
@@ -65,18 +79,46 @@ class ExplorationActivityPresenter @Inject constructor(
     this.internalProfileId = internalProfileId
     this.topicId = topicId
     this.storyId = storyId
+    this.explorationId = explorationId
     this.context = context
     this.backflowScreen = backflowScreen
+    if (getExplorationManagerFragment() == null) {
+      val explorationManagerFragment = ExplorationManagerFragment()
+      val args = Bundle()
+      args.putInt(
+        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY,
+        internalProfileId
+      )
+      explorationManagerFragment.arguments = args
+      activity.supportFragmentManager.beginTransaction().add(
+        R.id.exploration_fragment_placeholder,
+        explorationManagerFragment,
+        TAG_EXPLORATION_MANAGER_FRAGMENT
+      ).commitNow()
+    }
 
-    if (getExplorationFragment() == null) {
+  }
+
+  fun loadExplorationFragment(result: StoryTextSize) {
+    if (getExplorationManagerFragment() != null) {
       val explorationFragment = ExplorationFragment()
       val args = Bundle()
-      args.putInt(ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY, internalProfileId)
+      args.putInt(
+        ExplorationActivity.EXPLORATION_ACTIVITY_PROFILE_ID_ARGUMENT_KEY,
+        internalProfileId
+      )
       args.putString(ExplorationActivity.EXPLORATION_ACTIVITY_TOPIC_ID_ARGUMENT_KEY, topicId)
       args.putString(ExplorationActivity.EXPLORATION_ACTIVITY_STORY_ID_ARGUMENT_KEY, storyId)
-      args.putString(ExplorationActivity.EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY, explorationId)
+      args.putString(
+        ExplorationActivity.EXPLORATION_ACTIVITY_STORY_DEFAULT_FONT_SIZE_ARGUMENT_KEY,
+        result.name
+      )
+      args.putString(
+        ExplorationActivity.EXPLORATION_ACTIVITY_EXPLORATION_ID_ARGUMENT_KEY,
+        explorationId
+      )
       explorationFragment.arguments = args
-      activity.supportFragmentManager.beginTransaction().add(
+      activity.supportFragmentManager.beginTransaction().replace(
```

Comment:
Why this change? We shouldn't replace since we'll lose the bundle the fragment uses to retain its state on configuration changes.

Reply:
I faced some issue previously , not able to recollect what was that, now when i'am checking after revering this change its working fine. so i reverted back to add()